user/serializers.py

```diff
@@ -22,3 +23,39 @@ def update(self, instance, validated_data):
             user.save()
 
         return user
+
+
+class AuthTokenSerializer(serializers.Serializer):
+    """Serializer for user authentication object"""
+
+    email = serializers.CharField(label=_("Email"))
+    password = serializers.CharField(
+        label=_("Password", ),
+        style={'input_type': 'password'},
+        trim_whitespace=False
+    )
+
+    def validate(self, attrs):
+        """Validate and authentiate the user"""
+
+        email = attrs.get('email')
+        password = attrs.get('password')
+
+        user = authenticate(request=self.context.get('request'), username=email, password=password)
```

Comment:
Redundant code. You make user = authenticate here and in "if email and password", just remove this string.